core/lib/types/Cargo.toml

```diff
@@ -21,8 +21,8 @@ zksync_config = { path = "../config" }
 codegen = { git = "https://github.com/matter-labs/solidity_plonk_verifier.git", branch = "dev" }
 zkevm_test_harness = { git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.3.3" }
 zk_evm_1_4_1 = { package = "zk_evm", git = "https://github.com/matter-labs/era-zk_evm.git", branch = "v1.4.1" }
```

Comment:
Do you still need `zk_evm_1_4_1` though?

Reply:
@popzxc Unfortunately we need it, because `commitment_utils` need to convert the "general" `LogQuery` to each VM-specific one and so we still need to import those here to implement `From`

Reply:
The alternative is to make implement the conversion via `GlueFrom` + make it public + import it from `multivm`

Reply:
I'll tackle this once this get merged then.